widgets/contact/contact.php

```diff
@@ -1060,3 +1033,13 @@ static function sanitize_header( $value ){
 
 }
 siteorigin_widget_register( 'sow-contact-form', __FILE__, 'SiteOrigin_Widgets_ContactForm_Widget' );
+
+// Tell the autoloader where to look for contactform field classes.
+function contactform_fields_class_paths( $class_paths ) {
+	$class_paths[] = plugin_dir_path( __FILE__ ) . 'fields/';
+	return $class_paths;
+}
+add_filter( 'siteorigin_widgets_field_class_paths', 'contactform_fields_class_paths' );
```

Comment:
Re: my previous comment, this is the main issue with sharing a class loader between the base form fields and the contact form fields.They're sharing this same filter which implies they're the same thing.